cli/command/system/info_test.go

```diff
@@ -127,6 +127,12 @@ var sampleInfoNoSwarm = system.Info{
 			Size: 24,
 		},
 	},
+	FirewallBackend: &system.FirewallInfo{
+		Driver: "nftables+firewalld",
+		Info: [][2]string{
+			{"ReloadedAt", "2025-07-16T16:59:14Z"},
+		},
```

Comment:
I missed when we added this, but was there a specific reason for choosing `[][2]string` instead of `map[string]string`?

Reply:
Ah, that makes sense now, thanks for the context!